Lib/test/test_funcattrs.py

```diff
@@ -478,6 +479,27 @@ def test_builtin__qualname__(self):
         self.assertEqual([1, 2, 3].append.__qualname__, 'list.append')
         self.assertEqual({'foo': 'bar'}.pop.__qualname__, 'dict.pop')
 
+    @support.cpython_only   # See gh-58211
+    def test_builtin__self__(self):
+        import time
+        import builtins
+
+        # builtin function:
+        self.assertEqual(len.__self__, builtins)
```

Comment:
Use more strict assertion `assertIs` where it is possible.

Reply:
I'll do that.

But concerning the imports, should I keep locally or put globally?

Reply:
Either is fine. Thanks.